src/v3/src/transformer/layout/idxTransformerMapping.ts

```diff
@@ -136,6 +137,13 @@ const TransformerMap: {
     },
   },
   [IDX_STEP.CHALLENGE_AUTHENTICATOR]: {
+    [AUTHENTICATOR_KEY.SYMANTEC_VIP]: {
+      transform: transformSymantecVipAuthenticator,
+      buttonConfig: {
+        showDefaultSubmit: false,
+        showVerifyWithOtherLink: true,
```

Comment:
If I remember correctly, this is `true` by default, so you can remove this and it should work as expected.

Reply:
done